src/types/api/cluster.ts

```diff
@@ -36,12 +34,6 @@ export interface TClusterInfoV1 {
 
     /** Cluster root database */
     Domain?: string;
-
-    Balancer?: string; // additional
-    Solomon?: string; // additional
-
-    TraceView?: TTraceView;
-    TraceCheck?: TTraceCheck;
```

Comment:
These fields aren't actually present in `viewer/cluster` response, they were added while parsing `meta/cluster`